tqdm/tests/tests_version.py

```diff
@@ -4,7 +4,8 @@
 def test_version():
     """ Test version string """
     from tqdm import __version__
-    Mmpe = re.split('[.-]', __version__)
+    Mmpe = __version__.split()[0]  # remove part after space (branch name)
+    Mmpe = re.split('[.-]', Mmpe)  # split by dot and dash
     assert 3 <= len(Mmpe) <= 4
```

Comment:
Is this not the same as `assert len(Mmpe) in (3, 4)`? People usually expect reversed syntax, e.g. `4 >= len(Mmpe) >= 3`.